src/libraries/Common/src/System/Net/Security/NegotiateStreamPal.PlatformNotSupported.cs

```diff
@@ -0,0 +1,122 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Principal;
+using System.Text;
+using System.Threading;
+using Microsoft.Win32.SafeHandles;
+
+namespace System.Net.Security
+{
+    //
+    // The class maintains the state of the authentication process and the security context.
+    // It encapsulates security context and does the real work in authentication and
+    // user data encryption with NEGO SSPI package.
+    //
+    internal static partial class NegotiateStreamPal
```

Comment:
Why are these not annotated with Unsupported attribute

Reply:
Addressed, thanks